ushahidi/platform-client-mzima
# [Outreachy Task Submission] Make Icon-Only Buttons and Clickable Cards Accessible By Screen Readers

This pr aims to address [4756](https://github.com/ushahidi/platform/issues/4756).

**Description**
Currently some icon-only buttons and clickable cards on the Ushahidi Platform are not accessible by screen readers, so when a user tabs to that button or card location the screen reader just announces "button". This behaviour can confuse a user that relies heavily on keyboard for navigation as they won't know what the button or card does. I have attached a screen recording of the issue for better understanding.

**Before Fix**

https://github.com/ushahidi/platform-client-mzima/assets/46582086/2028f523-46ec-45a7-bb73-f509e0893d2b

**After Fix**

https://github.com/ushahidi/platform-client-mzima/assets/46582086/e25882f2-146f-41cc-9f99-827395a4f1a1

**How To Test**
1. Install [google chrome screen reader extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn)
2. Turn on the screen reader extension
3. Navigate through the Ushahidi data view page by pressing the tab key on your keyboard.
4. Observe that it pronounces the right button actions when the icon buttons receive focus.
